aeriscloud/cli/cloud/organization.py

```diff
@@ -166,6 +167,29 @@ def init(name, repository):
     """
     Initialize a new organization.
     """
+    if (not repository and
+            config.has('github', 'enabled') and
+            config.get('github', 'enabled') == 'true' and
+            config.has('github', 'token')):
+        gh = Github()
+
+        if '/' in name:
+            split = name.split('/')
```

Comment:
![screen shot 2016-04-05 at 11 02 24 am](https://cloud.githubusercontent.com/assets/29782/14269287/f0978916-fb1d-11e5-9571-a76847810a11.png)
Something like that?